examples/00_basic/example_01_simple_structural_solve.py

```diff
@@ -0,0 +1,212 @@
+""".. _ref_example_01_simple_structural_solve:
+
+Static structural analysis
+--------------------------
+
+Using supplied files, this example shows how to insert a static structural
+analysis into a new Mechanical session and execute a sequence of Python scripting
+commands that define and solve the analysis. Deformation results are then reported.
+
+"""
+
+###############################################################################
+# Download required files
+# ~~~~~~~~~~~~~~~~~~~~~~~
+# Download the required files. Print the file path for the geometry file.
+import os
+
+from ansys.mechanical.core import launch_mechanical
+from ansys.mechanical.core.examples import download_file
+
+geometry_path = download_file("example_01_geometry.agdb", "pymechanical", "00_basic")
+print(f"Downloaded the geometry file to: {geometry_path}")
+
+###############################################################################
+# Launch Mechanical
+# ~~~~~~~~~~~~~~~~~
+# Launch a new Mechanical session in batch, setting ``cleanup_on_exit`` to
```

Comment:
```suggestion
# Launch a new Mechanical session in batch, setting ``cleanup_on_exit`` argument to
```Goggle developer doc style guide says to follow something tagged as code with a noun descriptor.